.travis.yml

```diff
@@ -1,7 +1,7 @@
-language: python
+language: node_js
 sudo: false
-python:
-- '2.7'
+node_js:
+- '10'
 install:
 - nvm install 'lts/*'
```

Comment:
these 3 lines can go now 😄 nvm is a version manager for node. and `npm install npm@latest -g` was because of a  bug in npm years ago.

Reply:
To be clear, you mean these 3:

- nvm install 'lts/*'
- nvm use 'lts/*'
- npm install npm@latest -g

Also, they are redundant anyway, I think, since I switched the language to node_js and told it which version to use...

Reply:
yeah exactly 😄